worlds/cv64/client.py

```diff
@@ -0,0 +1,168 @@
+from typing import TYPE_CHECKING, Optional, Set
+from .locations import base_id
+from .text import cv64_text_wrap, cv64_string_to_bytes
+
+from NetUtils import ClientStatus
+import worlds._bizhawk as bizhawk
+from worlds._bizhawk.client import BizHawkClient
+
+if TYPE_CHECKING:
+    from worlds._bizhawk.context import BizHawkClientContext
+else:
+    BizHawkClientContext = object
```

Comment:
```py
else:
    BizHawkClientContext = object
```
Should be removed, and type hints for `BizHawkClientContext` should be wrapped in double quotes.